package.json

```diff
@@ -25,11 +25,17 @@
     "jest-environment-uint8array": "^1.0.0",
     "jest-mock-extended": "^1.0.10",
     "lerna": "^3.22.1",
+    "prettier": "^2.2.1",
     "ts-jest": "^26.4.4",
     "ts-node": "^9.0.0",
     "typedoc": "^0.19.2",
     "typedoc-plugin-lerna-packages": "^0.3.1",
     "typedoc-plugin-pages": "^1.0.1",
     "typescript": "^4.0.5"
+  },
+  "prettier": {
+    "printWidth": 120,
+    "trailingComma": "all",
+    "singleQuote": true
```

Comment:
Would be nice if we could start using the https://github.com/3box/eslint-config-3box package for this entire repo. But that is likely a much larger change.

Reply:
It definitely is bigger. 3box eslint rules are too strict for this code 😉